Bugzilla/WebService/Bug.pm

```diff
@@ -128,7 +129,15 @@ BEGIN {
 ###########
 
 sub fields {
-    my ($self, $params) = validate(@_, 'ids', 'names');
+    state $check = compile(
+        Object,
+        Dict[
+            ids   => Optional[ArrayRef[Int]],
```

Comment:
All the places we called validate, ArrayRef[Int] needs a coercion from Int
(and ArrayRef[Str] from Str)